language/Portuguese_Brazil/strings.po

```diff
@@ -2955,7 +2986,7 @@ msgstr "Remover dos Favoritos"
 #, c-format
 msgctxt "menu-main"
 msgid "About %1$s"
-msgstr "Sobre o %1$s"
+msgstr "Sobre o %s"
```

Comment:
Please don’t strip out SUS format argument index specifiers.

Reply:
This still needs to be corrected – please leave the SUS argument index specifier in place.